android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -169,8 +170,9 @@ public void onStartInput(EditorInfo attribute, boolean restarting) {
       if (kbInfo != null) {
         String langId = kbInfo.getLanguageID();
         SharedPreferences prefs = appContext.getSharedPreferences(appContext.getString(R.string.kma_prefs_name), Context.MODE_PRIVATE);
-        boolean mayPredict = prefs.getBoolean(KMManager.getLanguagePredictionPreferenceKey(langId), true);
-        KMManager.setBannerOptions(mayPredict);
+        int maySuggest = prefs.getInt(KMManager.getLanguageAutoCorrectionPreferenceKey(langId), KMManager.KMDefault_Suggestion);
+        // Enable banner if maySuggest is not SuggestionType.SUGGESTIONS_DISABLED (0)
+        KMManager.setBannerOptions(maySuggest > SuggestionType.SUGGESTIONS_DISABLED.toInt());
```

Comment:
```suggestion
        KMManager.setBannerOptions(maySuggest <> SuggestionType.SUGGESTIONS_DISABLED.toInt());
```

Yeah same same but removes knowledge of actual values :)

Reply:
yep. will use `!=`